rdflib/plugins/parsers/nquads.py

```diff
@@ -7,7 +7,7 @@
 >>> g = ConjunctiveGraph()
 >>> data = open("test/data/nquads.rdflib/example.nquads", "rb")
 >>> g.parse(data, format="nquads") # doctest:+ELLIPSIS
-<Graph identifier=... (<class 'rdflib.graph.Graph'>)>
+<Graph identifier=... (<class 'rdflib.graph.ConjunctiveGraph'>)>
```

Comment:
The doctest is updated to the correct class type now that ConjunctiveGraph's parse method returns Self.